src/store/selectors/selectors.ts

```diff
@@ -2,28 +2,26 @@ import { Ifilters, SelectedPhoto } from '../slices/gallery';
 import { RootState } from '../slices/types';
 import { PhotosData } from '../../app/types/photos-data';
 
-export const filtersSelector = (state: RootState): Ifilters =>
+const filtersSelector = (state: RootState): Ifilters =>
```

Comment:
use createSelector from reselect lib

Reply:
w jakim celu użyć tego createSelector? Co chcemy przez to uzyskać / optymalizować ?